docs/3_finetuner_in_action/3_3_text_to_image.md

```diff
@@ -1,37 +1,39 @@
 (text-to-image)=
-# Text to image search using CLIP
+# CLIP for text to image search
 
 This guide will showcase fine-tuning a `CLIP` model for text to image retrieval.
 
-## Task overview
+## Task
 We'll be fine-tuning CLIP on a fashion captioning dataset which contains information about fashion products.
```

Comment:
on the [fashion captioning dataset ](https://github.com/xuewyang/Fashion_Captioning)